crates/ruff_python_ast/src/nodes.rs

```diff
@@ -2516,33 +2516,10 @@ impl Parameters {
     }
 }
 
-#[allow(clippy::borrowed_box)] // local utility
```

Comment:
Okay, okay. And I deleted this unused code 😅

Reply:
Impressive...